src/app/workshop-details/workshop-details.component.html

```diff
@@ -63,13 +63,20 @@ <h2 class="highlight-headings">Price</h2>
                 <h2 class="highlight-headings">Workshop Link(s)</h2>
                 <div class="dates-wrapper">
                     <div class="dates-information" *ngFor="let detail of workshopDetails.multiWorkshopDetails;">
-                        <a href="{{detail.externalLink}}" (click)="clickExternalLink(detail.link)">
+                        <a title="Details for workshop starting on {{detail.startDateStr}}" href="{{detail.externalLink}}" (click)="clickExternalLink(detail.link)">
                             <i class="fa fa-calendar"></i>
                             <span>{{detail.startDateStr}} - {{detail.endDateStr}}</span>
                         </a>
                     </div>
                 </div>
             </div>
+            <div class="wsd-share">
```

Comment:
why did you add this?

Reply:
why couldn't we just show the navbar one and pass the url which we want?

Reply:
discussed offline